aten/src/ATen/native/native_functions.yaml

```diff
@@ -3673,9 +3673,13 @@
   variants: method, function
 
 - func: nonzero(Tensor self, *, Tensor(a!) out) -> Tensor(a!)
+  cpu_bool: True
```

Comment:
does this have any effect?  I thought this would only matter if these defined backend-specific dispatch, but neither this nor the one below it does.  Can you check?

Reply:
Yeah, you are right, this is not needed. Removed